src/AppBundle/Services/GuzzleClient.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace AppBundle\Services;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\TransferException;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+class GuzzleClient
```

Comment:
idea to have one service for all social networks. what about SocialUserProvider?